config/jobs/kubernetes/sig-network/ingress-gce-e2e.yaml

```diff
@@ -186,6 +186,9 @@ periodics:
       - --env=GCE_GLBC_IMAGE=gcr.io/k8s-ingress-image-push/ingress-gce-glbc-amd64:nightly
       - --env=GCE_ALPHA_FEATURES=NetworkEndpointGroup
       - --env=KUBE_GCE_ENABLE_IP_ALIASES=true
+      - --env=KUBE_FEATURE_GATES=DisableCloudProviders=true,DisableKubeletCloudCredentialProviders=true
```

Comment:
this is from

https://github.com/kubernetes/test-infra/blob/3431b4e1e123f00e1a8ee8ee9e60a7610b25304f/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L489-L505